mne/viz/_3d.py

```diff
@@ -1344,3 +1346,141 @@ def _get_view_to_display_matrix(scene):
                                  [0.,            0.,   1.,        0.],
                                  [0.,            0.,   0.,        1.]])
     return view_to_disp_mat
+
+
+def plot_dipole_mri_orthoview(dipole, trans, subject, subjects_dir=None,
+                              scale_factor=1e9, ax=None, block=False,
+                              show=True):
+    """Plot dipoles on top of mri slices in 3-D.
+
+    Browse through the dipoles using mouse scroll or up/down arrows.
+
+    Parameters
+    ----------
+    dipole : instance of mne.Dipole
+        The dipole to plot.
+    trans : dict
+        The mri to head trans.
+    subject : str
+        The subject name corresponding to FreeSurfer environment variable
+        SUBJECT.
+    subjects_dir : None | str
+        The path to the freesurfer subjects reconstructions. It corresponds to
+        Freesurfer environment variable SUBJECTS_DIR. The default is None.
+    scale_factor : float
+        The scaling applied to convert amplitudes to vector (arrow) lengths for
+        the plot. Defaults to 1e9.
```

Comment:
I know it's the scale factor for the amplitude, that was clear enough already. What isn't clear is the units. I think it should say something like "... to convert from amplitude in Am to arrow length in mm, e.g., 1e9 (default) makes a 10 nAm dipole appear with a length of 10 mm in the plot" (if this is indeed what the conversion factor is)